ceph-salt-formula/salt/_modules/ceph_salt.py

```diff
@@ -65,3 +66,10 @@ def probe_ntp(ahost):
         return 1
     except:
         return 3
+
+def is_safety_disengaged():
+    execution = __pillar__['ceph-salt'].get('execution', {})
+    safety_disengage_time = execution.get('safety_disengage_time')
+    if not safety_disengage_time or safety_disengage_time + 60 <= time.time():
+        return False
+    return True
```

Comment:
could we reverse the logic here to make it easier for my poor brain to comprehend?

```
    if safety_disengage_time and safety_disengage_time + 60 <= time.time():
        return True
    return False
```

This makes it more obvious that the function defaults to returning "False" and only returns True under certain strict conditions.

Reply:
Done